frontend/app_flowy/pubspec.yaml

```diff
@@ -45,6 +45,10 @@ dependencies:
       ref: dbc309f5e382963fa0122409a0aaf8e1417d51c1
 
   #  third party packages
+  boardview:
+    git:
+      url: git://github.com/hieugao/boardview.git
```

Comment:
Any issues that prevent using the pub.dev version?

Reply:
I think push to pub.dev would be better.

Reply:
I'm against using a package that's not in active development. Better to implement it ourselves in that case

Reply:
That's why I said it will be included into AppFlowy.